contents/blog/cypress-end-to-end-tests.md

```diff
@@ -0,0 +1,233 @@
+---
+date: 2020-06-10
+title: Merge with Confidence
```

Comment:
I wonder what the best title is here. Maybe something like "how to set up super fast Cypress tests on Github Actions?"

Reply:
I'd vote for the above unless any other ideas